dhis-2/dhis-support/dhis-support-artemis/src/main/java/org/hisp/dhis/artemis/MessageManager.java

```diff
@@ -41,11 +41,13 @@
 public class MessageManager
 {
     private final JmsTemplate jmsTemplate;
+    private final JmsTemplate jmsQueueTemplate;
     private final RenderService renderService;
 
-    public MessageManager( JmsTemplate jmsTemplate, RenderService renderService )
+    public MessageManager( JmsTemplate jmsTemplate, JmsTemplate jmsQueueTemplate, RenderService renderService )
```

Comment:
Can we rename `JmsTemplate` to `JmsTopicTemplate` so that it's clear what each template is for?

Reply:
make sense.